defaults/main.yml

```diff
@@ -59,7 +59,7 @@ logstash_major_ver: '5.x'  # Define major version to install
 # Defines the miniumum amount of memory (in MB) required to effectively run Logstash
 logstash_min_memory_required: 1024
 
-logstash_minor_ver: '1:5.2.0-1'  # Define minor version to install
```

Comment:
I understand why it was added in this way, but doesn't works in debian based like ubuntu because: 

If you upgrade `apt-get upgrade` it will upgrade the installed version with `apt`. Then you run the role and it fails saying: it can't downgrade the version. 

So another alternative is to use this var and set an `apt-policy` to ensure the version defined in this var will be keep even when using `apt-get upgrade`. 

If you prefer to go in the way of `apt-policy` I can spent some time on fixing it in that way.